pandas/core/sparse/series.py

```diff
@@ -256,8 +256,8 @@ def npoints(self):
     def from_array(cls, arr, index=None, name=None, copy=False,
                    fill_value=None, fastpath=False):
         """
-        DEPRECATED: use the pd.SparseSeries(..) constructor instead.
-
```

Comment:
can you add the 1-liner description here (IOW the deprecated should not be the first line)